src/main/java/application/controllers/TreeController.java

```diff
@@ -131,14 +130,15 @@ public void colorSelectedStrains() {
      */
     public void applyCellHighlight(Cell cell) {
         if (cell instanceof LeafCell) {
-            List<Cell> parentList = new ArrayList<>();
-            parentList.add(cell);
+            String temp = ((LeafCell) cell).getName();
             collectedStrains.clear();
-            collectedStrains.add(cell);
 
-            applyColorUpwards(parentList,
-                    determineLinColor(metaData.get(((LeafCell) cell).getName())),
-                    4.0);
+            if (temp.contains("TKK")) {
```

Comment:
Paths to/from the following samples are no longer highlighted:
- reference
- G51950
- G51951
- G51952